server/test/services/applicant/ReadOnlyApplicantProgramServiceImplTest.java

```diff
@@ -124,14 +124,7 @@ public void getStoredFileKeys_worksForMultipleFileUploads() {
     QuestionAnswerer.answerFileQuestionWithMultipleUpload(
         applicantData,
         ApplicantData.APPLICANT_PATH.join(fileQuestionDefinition.getQuestionPathSegment()),
-        0,
-        "file-key");
-
-    QuestionAnswerer.answerFileQuestionWithMultipleUpload(
-        applicantData,
-        ApplicantData.APPLICANT_PATH.join(fileQuestionDefinition.getQuestionPathSegment()),
-        1,
-        "file-key-2");
+        ImmutableList.of("file-key", "file-key-2"));
```

Comment:
**praise:** I'm loving that you're reducing all these lines of code down to 1.  So satisfying!